site/content/entry/telemetry-and-why-thats-a-good-thing.md

```diff
@@ -0,0 +1,41 @@
++++
+date = "2017-08-02T21:02:00-04:00"
+title = "Telemetry (And Why That's A Good Thing)"
+tags = [ "citra-release" ]
+author = "anodium"
+forum = 0000
++++
+
+Citra has some [issues](https://github.com/citra-emu/citra/issues), we've always been transparent in what they are, and have always allowed anyone to contribute to it. Unfortunately though, most contributions are made by [a small minority of developers](https://github.com/citra-emu/citra/graphs/contributors). These developers have found it difficult to prioritize their efforts, since the issues are mostly reported outside of the GitHub tracker, on the forums, on chats, and elsewhere. Not to mention that most of the reports are too vague, are missing vital information such as the logs or system info, or both.
+
+Because of this, the Citra team has put together a framework to report data about our users to our server, and use that data to find things like where emulated games crash in Citra most often, what are the most popular games and hardware configurations, and so on. We had considered including this in last month's progress report, but we decided that it would be best to publish this on its own, so it gets the attention it deserves. We're very aware that privacy is important to many of our users, and so from the earliest planning stage we knew we must be as transparent and open about this as possible.
```

Comment:
"find things like" can be changed to "discover" - "and use that data to discover what the most popular games and hardware configurations are, as well as where emulated games crash most often in Citra, and more."